packages/blade/src/components/Checkbox/CheckboxGroup/CheckboxGroup.tsx

```diff
@@ -112,7 +126,7 @@ const CheckboxGroup = ({
           {label}
         </FormLabel>
         <Box>
-          <Box display="flex" flexDirection="column" gap={2}>
+          <Box display="flex" flexDirection="column" gap={gap}>
```

Comment:
`gap` doesn't work on react native. Can we remove gap's usage while we're working on this PR? I'm guessing this renders incorrectly on react native because of this @anuraghazra